lib/client.js

```diff
@@ -87,7 +87,7 @@ export default class Client extends EventEmitter {
     if (config.method === undefined) config.method = 'GET';
     if (!(config.url instanceof URL)) config.url = new URL('' + config.url, this.baseURL);
     if (config.headers === undefined) config.headers = {};
-    if (config.websocket === true && config.protocols !== undefined) config.protocols = [];
+    if (config.websocket && config.protocols === undefined) config.protocols = [];
```

Comment:
Why move away from a type check?

Reply:
I'm so used to allowing truthy things to mean truthy that to me checking that something is the actual true object feels like a code smell. In this case since you are setting the literal true explicitly elsewhere I guess it is fine.